src/templates/components/ManualCard.svelte

```diff
@@ -1,50 +1,40 @@
-<script context="module" lang="ts">
-	import { CLICK_MACRO } from '$lib/gam';
-	import type { GAMVariable } from '$lib/gam';
-</script>
-
 <script lang="ts">
 	import '$templates/components/fonts/Egyptian.css';
 	import '$templates/components/fonts/Headline.css';
 	import '$templates/components/fonts/Sans.css';
 	import ArrowRight from './icons/ArrowRight.svelte';
 
-	export let TotalCardNumber: number;
-	export let EventTitle: GAMVariable;
-	export let EventDateTime: GAMVariable;
-	export let EventImage: GAMVariable;
-	export let EventUrl: GAMVariable;
+	export let image: string;
+	export let url: string;
+	export let callToAction: string;
+	export let buttonColour: string;
 	export let direction = 'row';
-
-	let [boldTitle, regularTitle] = EventTitle.split(':');
-	if (regularTitle) {
-		boldTitle += ':';
-	} else {
-		regularTitle = '';
-	}
+	export let isProminent = false;
 </script>
 
 <a
-	class="card split-into-{TotalCardNumber}"
```

Comment:
The split into stuff didn't actually work, seems better to filter the events that actually have text in them https://github.com/guardian/commercial-templates/pull/338/files#diff-e5ba83f56a29bdc63ad7241c5b53c92078a77913d1af94896e4a83c974231bf6R69